searchsummary/src/vespa/searchsummary/docsummary/getdocsumargs.h

```diff
@@ -31,15 +32,14 @@ class GetDocsumArgs
     void SetRankProfile(const vespalib::string &ranking) { _ranking = ranking; }
     void setResultClassName(vespalib::stringref name) { _resultClassName = name; }
     void SetStackDump(uint32_t stackItems, uint32_t stackDumpLen, const char *stackDump);
-    void setLocation(vespalib::stringref location) {
-        _location = location;
-    }
-
+    void no_locations(bool value) { _no_locations = value; }
```

Comment:
Consider rename to "has_locations", as I think the code using this is easier to read that way.

Reply:
that doesn't really capture the semantics; it is a flag that is set if we tried to parse locations but didn't find anything useful, to avoid having to do that multiple times.  if we want a positive flag it would have to be something like "locations_possible" - will implement that to see if it reads better.